binaries/scheduler/main.go

```diff
@@ -29,12 +28,15 @@ type servers struct {
 	thrift thrift.TServer
 	http   *endpoints.TwitterServer
 	sched  scheduler.Scheduler
-	queue  queue.Queue
 	stat   stats.StatsReceiver
 }
 
-func makeServers(thrift thrift.TServer, http *endpoints.TwitterServer, sched scheduler.Scheduler, queue queue.Queue, stat stats.StatsReceiver) servers {
-	return servers{thrift, http, sched, queue, stat}
+func makeServers(
+	thrift thrift.TServer,
```

Comment:
Do you still need all of these args? e.g., do you still need the scheduler if the http server has a reference to it?